File: challenges/02_CalcularElVolumenDeUnCilindro/main.c
#include <stdio.h>
#include <stdlib.h>

void main() {
  float radio, altura;

  printf("Reto 02:Calcular el volumen de un cilindro. \n\n");
  printf("Ingresa el radio de la base: ");
  scanf("%f", &radio);

  printf("Ingresa la altura del cilindro: ");
  scanf("%f", &altura);

  printf("\nEl volumen del cilindro es: %f", (radio * 3.1416) * altura);
}

File: challenges/02_CalcularElVolumenDeUnCilindro/main.py
def main():

    radio = int(input("Ingresa el radio de la base: "))

    area = radio * 3.1416

    altura = int(input("Ingresa la altura de la base: "))

    volumen = area * altura

    print("El volumen del cilindro es " + str(volumen))


if __name__ == "__main__":
    main()
